data/about.yml

```diff
@@ -3,13 +3,13 @@ heading : Our
 headingSpan : Services
 aboutItem :
 - icon : tf-tools
-  title : Cloud Native consultancy
-  content : We're consulting companies in transition to cloud and agile infrastructure.
+  title : Cloud Native Consultancy
+  content : We consult companies on their cloud native transition, agile infrastructure, and DevOps best practices. 
```

Comment:
Looks good, maybe ‘we consult with companies...’

Reply:
done